tests/variable_state/bad/no_cross_exceptions.catala_en

```diff
@@ -16,8 +16,7 @@ scope A:
 
 ```catala-test-inline
 $ catala Typecheck 
-[ERROR]
-Unknown label for the scope variable foo@baz: "thing"
+[ERROR] Unknown label for the scope variable foo@baz: "thing"
```

Comment:
Maybe replacing `foo@baz` by `foo state baz` (localized depending on the language) would be better.

Reply:
It'd definitely be more readable, yes, I'll see what can be done.